conteudo/introducao/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Introdução
+description: Como Instalar e deixar o Brigade funcionando
+section: introducao
+aliases:
```

Comment:
I think it would be better to write the header in Portuguese, what do you think?

```suggestion
título: Introdução
descrição: Como Instalar e deixar o Brigade funcionando
sessão: introdução
aliases:
```

Reply:
fixed. I am keeping metadata titles in English.